trigger-actions-framework/main/default/objects/sObject_Trigger_Setting__mdt/fields/TriggerRecord_Class_Name__c.field-meta.xml

```diff
@@ -5,7 +5,7 @@
         and `recordPrior`. Both of these variables need to match the SObject type for this SObject
         Trigger Setting record.</description>
     <externalId>false</externalId>
-    <fieldManageability>SubscriberControlled</fieldManageability>
```

Comment:
Let's keep this as the same for now.

Reply:
That was my intention - apologies; this slipped through a crack.